code-challenges.test.js

```diff
@@ -43,26 +33,36 @@
 
 // b) Create the function that makes the test pass.
 // Psuedo Code:
-// For this I need to create a function that takes in the variable "fibonacciSequence"
-// I'm going to utilize the for loop method to add up the numbers
-// I'll create two extra variables of 0 and 1 
-
+// Create a function called fibonacciSequence with a num param
+// Create a variable with an array of 1,1 to hold the first two values of the Fibonacci Sequence
+// Create a for loop so the function will increase with each iteration
+// Use .push to push the index of the array variable and + fSArray[i -1] 
+// return fSArray
+
+const fibonacciSequence = (number) => {
+  let fSArray = [1, 1]
+  for(let i = 1; i < number-1; i++) {
+    fSArray.push(fSArray[i] + fSArray[i - 1])
+  }
+  return fSArray
+}
+// Pass:    ✓ takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence (1 ms)
 
 // --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-// describe("oddArray", () => {
-//     const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
-//     const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
-//     const oddArray = (array) => {
-//         return array.filter(value => value % 2 !== 0)
-//     }
-//     it("returns a new array of only odd numbers sorted from least to greatest.", () => {
-//       expect(oddArray(fullArr1)).toEqual([-9, 7, 9, 199])
-//       expect(oddArray(fullArr2)).toEqual([-823, 7, 23])
-//     })
-//   })
+describe("oddArray", () => {
+    const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
+    const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
+    // const oddArray = (array) => {
+    //     return array.filter(value => value % 2 !== 0)
+    // }
+    it("returns a new array of only odd numbers sorted from least to greatest.", () => {
+      expect(oddArray(fullArr1)).toEqual([-9, 7, 9, 199])
+      expect(oddArray(fullArr2)).toEqual([-823, 7, 23])
+    })
+  })
```

Comment:
Test looks awesome.  I would recommend removing lines 58 - 60 since we don't want to put any logic into our tests.  Also, I like that you placed your test variables inside the test.  Oftentimes test variables are placed inside the it method just before your expect statement.